docs/fips.md

```diff
@@ -0,0 +1,195 @@
+# BoringCrypto and FIPS 140-3 compliance
+
+## Introduction
+
+Sumo OT distro can be built to use BoringCrypto instead of Go standard library crypto implementations.
```

Comment:
Maybe add something like "BoringCrypto is a cryptographic module validated against FIPS 140."

(Would it help to add a link to https://csrc.nist.gov/projects/cryptographic-module-validation-program/validated-modules/search?SearchMode=Basic&ModuleName=boring&CertificateStatus=Active&ValidationYear=0?)